website/docs/language/expressions/custom-conditions.mdx

```diff
@@ -203,6 +204,15 @@ You should also consider the following questions when creating preconditions and
 - Which approach is more convenient? If a particular resource has many dependencies that all make an assumption about that resource, it can be pragmatic to declare that once as a post-condition of the resource, rather than declaring it many times as preconditions on each of the dependencies.
 - Is it helpful to declare the same or similar conditions as both preconditions and postconditions? This can be useful if the postcondition is in a different module than the precondition because it lets the modules verify one another as they evolve independently.
 
+## Check Assertions
+
+-> **Note:** Check blocks and their assertions are only available in Terraform v1.5.0 and later.
+
+Add one or more `assert` blocks within a `check` block to specify custom conditions. Each assertion requires a [`condition` argument](#condition-expressions), an expression that must use the value of the variable to return `true` if the value is valid, or `false` if it is invalid. The expression can refer to any resource, data source, or variable available to the surrounding check block.
+
+If the condition evaluates to `false`, Terraform produces an [error message](#error-messages) that includes the result of the `error_message` expression. If you declare multiple assertions, Terraform returns error messages for all failed conditions.
+
+For more information on check blocks in general, refer to the Terraform language page for [Checks](/terraform/language/checks).
 
 ## Condition Expressions
```

Comment:
We should update the sentence on the line _right_ underneath this one to include "checks":

"Checks, input variable validation, preconditions, and postconditions all require a condition argument."

Reply:
Underneath this section is one named ["Conditions Checked Only During Apply",](https://developer.hashicorp.com/terraform/language/expressions/custom-conditions#conditions-checked-only-during-apply) does it need to be updated for Checks? It feels like yes, but wanted to see what you thought.